samples/bluetooth/connection_event_trigger/README.rst

```diff
@@ -0,0 +1,165 @@
+.. _ble_connection_event_trigger:
+
+Bluetooth: Event Trigger
+########################
+
+.. contents::
+   :local:
+   :depth: 2
+
+The Bluetooth® Event Trigger sample showcases the Event Trigger feature, a proprietary
+extension of the Bluetooth® protocol which can be used to schedule sampling, GPIO toggling, or other SW- or HW-based
+user activities in synchronization with individual BLE role instances, such as a central or peripheral link.
```

Comment:
```suggestion
The Bluetooth® Event Trigger sample showcases the Event Trigger feature.
It is a proprietary extension of the Bluetooth® protocol and can be used to schedule sampling, GPIO toggling, or other SW- or HW-based user activities in synchronization with individual |BLE| role instances, such as a central or peripheral link.
```

Reply:
This fails to build the docs:

/home/runner/work/sdk-nrf/sdk-nrf/ncs/nrf/doc/_build/nrf/src/samples/bluetooth/connection_event_trigger/README.rst:10: ERROR: Undefined substitution referenced: "BLE".

Reply:
Huh, I guess we only have the shortcut in nrfxlib. So `|BLE|` should be replaced with `Bluetooth LE`.